src/test/scala/ai/privado/languageEngine/ruby/download/ExternalDependenciesDownloadTest.scala

```diff
@@ -31,7 +31,8 @@ class ExternalDependenciesDownloadTest extends ExternalDependenciesDownloadTestB
     testGemMap.toMap
   }
 
-  "Ruby External Dependency Download" should {
```

Comment:
@khemrajrathore add TODO to fix this one.

Reply:
This will be eliminated and new test case will have to be written